daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/nested-separator-delimited.tdml

```diff
@@ -202,61 +202,14 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
-
-
-  <tdml:defineSchema name="nest1">
-    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
-    <dfdl:format ref="ex:GeneralFormat" occursCountKind="parsed" />
-
-    <xs:element name="nest1" dfdl:lengthKind="implicit"
-      dfdl:terminator="!">
-      <xs:complexType>
-        <xs:sequence dfdl:separator=";">
-          <xs:sequence dfdl:separator=",">
-            <xs:element name="a" type="xs:int" minOccurs="0"
-              maxOccurs="unbounded" dfdl:occursCountKind="parsed"
-              dfdl:lengthKind="delimited" />
-          </xs:sequence>
-        </xs:sequence>
-      </xs:complexType>
-    </xs:element>
-  </tdml:defineSchema>
-
-  <tdml:parserTestCase name="nest1" root="nest1"
-    model="nest1" description="Unambiguous nested separators, no terminators.">
-
-    <tdml:document>
-      <tdml:documentPart type="text"><![CDATA[0,1,2;3,4,5;6,7;8;9!]]></tdml:documentPart>
-    </tdml:document>
-
-    <tdml:infoset>
-      <tdml:dfdlInfoset xmlns:xs="http://www.w3.org/2001/XMLSchema"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-        <nest1>
-          <a>0</a>
-          <a>1</a>
-          <a>2</a>
-          <a>3</a>
-          <a>4</a>
-          <a>5</a>
-          <a>6</a>
-          <a>7</a>
-          <a>8</a>
-          <a>9</a>
-        </nest1>
-      </tdml:dfdlInfoset>
-    </tdml:infoset>
-  </tdml:parserTestCase>
```

Comment:
Is there any value in keeping this test but changing it to a negative test that show's that this kind of nesting of separators doesn't work, and that finding the outer separator will cause the array to end?

Reply:
I will update it to expect the error.